app/service/CasesService.scala

```diff
@@ -70,4 +70,12 @@ class CasesService @Inject()(connector: BindingTariffClassificationConnector, em
     }
   }
 
+  def addCaseCreatedEvent(atar: Case, operator: Operator)
+                                 (implicit hc: HeaderCarrier): Future[Unit] = {
+    val event = NewEventRequest(CaseCreated("Application submitted"), operator)
```

Comment:
Similar to what David suggested here https://github.com/hmrc/tariff-classification-frontend/pull/379#discussion_r477148443

It might be good to add/use the `addEvent` private function here as well (even though this will be the only consumer in this project for now, it makes it a bit clearer).